src/sentry/utils/event_frames.py

```diff
@@ -1,11 +1,13 @@
 from __future__ import annotations
 
+from collections import namedtuple
 from copy import deepcopy
 from typing import Any, Callable, Mapping, MutableMapping, Optional, Sequence, Tuple, cast
 
 from sentry.utils.safe import PathSearchable, get_path
 
 FrameMunger = Callable[[str, MutableMapping[str, Any]], bool]
+SdkFrameMunger = namedtuple("SdkFrameMunger", ["frame_munger", "requires_sdk", "supported_sdks"])
```

Comment:
Could potentially make this a dataclass so that we can define defaults and add types for each of these. Would also help us avoid things like `munger[1]` which is a little hard to follow

Reply:
Good idea!